crates/astria-composer/env.example

```diff
@@ -12,3 +12,6 @@ ASTRIA_COMPOSER_CHAIN_ID="912559"
 
 # Address of the RPC server for execution
 ASTRIA_COMPOSER_EXECUTION_URL="ws://127.0.0.1:8545"
+
+# Private key for the sequencer account used for signing transactions
+ASTRIA_COMPOSER_PRIVATE_KEY="2bd806c97f0e00af1a1fc3328fa763a9269723c8db8fac4f93af71db186d6e90"
```

Comment:
Does this private key correspond to one of the accounts in the local test genesis (would be good to have a funded account already here)?

Reply:
yeah it corresponds to the `alice` account